sse2neon.h

```diff
@@ -1709,19 +1709,16 @@ FORCE_INLINE __m128i _mm_srai_epi16(__m128i a, int imm)
 
 // Shift packed 64-bit integers in a left by imm8 while shifting in zeros, and
 // store the results in dst.
-#define _mm_slli_epi64(a, imm)                                   \
-    __extension__({                                              \
-        __m128i ret;                                             \
-        if ((imm) <= 0) {                                        \
-            ret = a;                                             \
-        } else if ((imm) > 63) {                                 \
-            ret = _mm_setzero_si128();                           \
-        } else {                                                 \
-            ret = vreinterpretq_m128i_s64(                       \
-                vshlq_n_s64(vreinterpretq_s64_m128i(a), (imm))); \
-        }                                                        \
-        ret;                                                     \
-    })
+FORCE_INLINE __m128i _mm_slli_epi64(__m128i a, int imm)
+{
+    if (imm <= 0) {
+        return a;
+    } else if (imm > 63) {
```

Comment:
Change `} else if (imm > 63) {` to `if (imm > 63) {`.

Reply:
I am curious about the reason for such a change.

Reply:
Shorten code and make it cleaner.

Reply:
got it

Reply:
Later, we can introduce macros such as `likely` and `unlikely` as the hint to the compiler to emit instructions that will cause branch prediction to favour the "likely" side of a jump instruction. 
See https://stackoverflow.com/questions/109710/how-do-the-likely-unlikely-macros-in-the-linux-kernel-work-and-what-is-their-ben

Reply:
Let me open another new issue for this requirement